code/__DEFINES/say.dm

```diff
@@ -37,8 +37,6 @@
 #define MODE_HOLOPAD "holopad"
 
 #define MODE_CHANGELING "changeling"
```

Comment:
Is this used anywhere?

Reply:
They were also related to ling chat, removed